src/main/java/uk/co/orchardsystems/cinema_ticketing/util/DataLoader.java

```diff
@@ -0,0 +1,84 @@
+package uk.co.orchardsystems.cinema_ticketing.util;
+
+import com.mongodb.*;
+import com.mongodb.util.JSON;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Properties;
+
+public class DataLoader {
+    private static Properties prop = loadProperties();
+
```

Comment:
Good Work. This could be reduced to a few lines if you used Spring Boot's libraries.